standards/RDA/ruleset.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="RDA">
+	<description>Reader's Digest Association WordPress Coding Standards</description>
+
+	<rule ref="WordPress-Core">
+		<!-- This rule is foolish and breaks many clean code uses -->
+		<exclude name="PEAR.Functions.FunctionCallSignature.Indent" />
+		<exclude name="WordPress.PHP.YodaConditions" />
+		<exclude name="WordPress.WP.GlobalVariablesOverride" />
+		<!-- This rule is foolish and breaks many legitimate uses -->
+		<exclude name="WordPress.Security.EscapeOutput.OutputNotEscaped" />
+		<exclude name="WordPress.Files.FileName" />
+		<exclude name="WordPress.WP.EnqueuedResources.NonEnqueuedScript" />
```

Comment:
Non-enqueued scripts can't be handled by concatenation and minification plugins, which has an impact in performance. Should we review use cases where this rule is a problem?